wp2static-addon-ftp.php

```diff
@@ -55,7 +55,6 @@
 function run_wp2static_addon_ftp() {
     $plugin = new WP2Static\FTPAddon();
     $plugin->run();
-
 }
 
-run_wp2static_addon_ftp();
+run_wp2static_addon_ftp();
```

Comment:
We should keep a newline at the end of each file as best practice. 

I pushed the swp file removal before this, so you should be able to pull latest and revise this with another commit

Reply:
Perfect!